packages/core/src/auth/tokens/token-scope.ts

```diff
@@ -1,26 +1,66 @@
 export enum CacheRole {
   None = 'none',
   ReadWrite = 'readwrite',
+  ReadOnly = 'readonly',
 }
 
+export class All {}
```

Comment:
I don't feel strongly about this but if you liked `Any` better, you could call this `AnyCache`.

Reply:
Allen pointed out Any has special meaning in JS. I am fine with All - this way I don't have to create two messages `AnyCache` and `AnyTopic`

Reply:
that's fine with me but there will probably be some language where `All` is a keyword :P

we can just give it a different name in that SDK though if/when he hit that :)